x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/utils.ts

```diff
@@ -9,15 +9,22 @@ import { cloneDeep, omit } from 'lodash';
 
 import { SchemaType } from '../../../shared/schema/types';
 
-import { RawBoost, Boost, SearchSettings, BoostType, ValueBoost } from './types';
+import {
+  RawBoost,
+  Boost,
+  SearchSettingsRequest,
+  SearchSettings,
+  BoostType,
+  ValueBoost,
+} from './types';
 
 // If the user hasn't entered a filter, then we can skip filtering the array entirely
 export const filterIfTerm = (array: string[], filterTerm: string): string[] => {
   return filterTerm === '' ? array : array.filter((item) => item.includes(filterTerm));
 };
 
-export const removeBoostStateProps = (searchSettings: SearchSettings): SearchSettings => {
-  const updatedSettings = cloneDeep(searchSettings);
+export const removeBoostStateProps = (searchSettings: SearchSettings): SearchSettingsRequest => {
```

Comment:
Remove `precision_enabled` from requests, make it explicit using the new interface